src/quo/components/wallet/account_overview/view.cljs

```diff
@@ -2,10 +2,12 @@
   (:require
     [quo.components.icon :as icons]
     [quo.components.markdown.text :as text]
+    [quo.components.wallet.account-overview.schema :as component-schema]
     [quo.components.wallet.account-overview.style :as style]
     [quo.foundations.colors :as colors]
     [quo.theme :as quo.theme]
     [react-native.core :as rn]
+    [schema.core :as schema]
```

Comment:
imo we should move this schema.core stuff to schema files and create the wrapper in there. We should only need one import into a view file related to schemas.
wdyt @ilmotta, @clauxx ?

Reply:
It could be a function that would wrap instrumentation, but it feels boilerplate-ish to have in each. Would be easier if we used `schema/=>`, then from the component-schema file we could just require the `view` and instrument it, but there was an issue with that that I don't remember now from our initial conversations.

Reply:
https://www.youtube.com/watch?v=_K-L9uhsBLM :)

Reply:
hauahahaha any small time make big time!